cypress/features/regression/confirmActions.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Confirm component - Actions
```

Comment:
Test cases in this file are not only about actions, so this name doesn't fit here. Maybe below will be better ? 
```suggestion
Feature: Confirm component - in Iframe
```